Core/Src/logic_layer.c

```diff
@@ -106,7 +180,7 @@ uint8_t r3g3b2_Colour(char command[])
  */
 int logic_layer(char commando[])
 {
-	char commando_list[NUMBER_OF_COMMANDS][MAX_NUMBER_OF_SCRIPT_CHARACTER] = {"lijn", "rechthoek", "tekst", "bitmap", "clearscherm", "cirkel", "figuur" }; // The script commando that the project needs to do.
+	char commando_list[NUMBER_OF_COMMANDS][MAX_NUMBER_OF_SCRIPT_CHARACTER] = {"lijn", "rechthoek", "tekst", "bitmap", "clearscherm", "cirkel", "figuur", "herhaal", "wait"}; // The script commando that the project needs to do.
```

Comment:
"wait" should be "wacht", same as in script.

Reply:
changed wait to wacht